config-store-yaml/pom.xml

```diff
@@ -0,0 +1,181 @@
+				<dependencies>
+					<dependency>
+						<groupId>org.junit.jupiter</groupId>
+						<artifactId>junit-jupiter-engine</artifactId>
+						<version>${junit.version}</version>
+					</dependency>
+				</dependencies>
+				<executions>
+					<execution>
+						<goals>
+							<goal>integration-test</goal>
+							<goal>verify</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<parallel>suites</parallel>
+					<threadCountSuites>1</threadCountSuites>
+					<reuseForks>false</reuseForks>
+					<forkCount>1</forkCount>
+				</configuration>
+			</plugin>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>org.junit.jupiter</groupId>
+						<artifactId>junit-jupiter-engine</artifactId>
+						<version>${junit.version}</version>
+					</dependency>
+				</dependencies>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>unpack-sjk</id>
+						<phase>compile</phase>
+						<goals>
+							<goal>unpack</goal>
+						</goals>
+						<configuration>
+							<artifactItems>
+								<artifactItem>
+									<groupId>org.gridkit.jvmtool</groupId>
+									<artifactId>sjk-core</artifactId>
+									<version>0.14</version>
+									<outputDirectory>${project.build.directory}/sjk-core</outputDirectory>
+								</artifactItem>
+							</artifactItems>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-antrun-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>fix-sjk-core</id>
+						<phase>process-classes</phase>
+						<goals>
+							<goal>run</goal>
+						</goals>
+						<configuration>
+							<tasks>
+								<!-- Remove root package classes -->
+								<delete>
+									<fileset dir="${project.build.directory}/sjk-core/" includes="*.class"/>
+								</delete>
+								<jar destfile="${project.build.directory}/classes/sjk-core-0.14.jar">
+									<fileset dir="${project.build.directory}/sjk-core"/>
+								</jar>
+								<delete dir="${project.build.directory}/sjk-core"/>
+							</tasks>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<configuration>
+					<unpackBundle>true</unpackBundle>
+					<instructions>
+						<Bundle-Name>Config-Store-Yaml</Bundle-Name>
+						<Bundle-Description>Config store YAML implementation</Bundle-Description>
+						<Bundle-SymbolicName>io.stargate.config.store.yaml</Bundle-SymbolicName>
+						<Bundle-Activator>io.stargate.config.store.yaml.ConfigStoreActivator</Bundle-Activator>
+						<Import-Package><![CDATA[
+              org.osgi.framework,
+              io.stargate.config.store,
```

Comment:
What do you think about moving API into `io.stargate.config.store.api`? If API and impl. are sibling packages it should make it easier to deal with package selectors, etc... it does seem to be an issue now, but might become prominent at the code evolves.